core/src/web/local-notifications.ts

```diff
@@ -133,7 +132,7 @@ export class LocalNotificationsPluginWeb
     });
   }
 
-  requestPermissions(): Promise<PermissionsRequestResult> {
+  requestPermissions(): Promise<{ results: [any] }> {
```

Comment:
`[any]` = 1-tuple whose element is of type `any`
`any[]` = array of `any`

```suggestion
  requestPermissions(): Promise<{ results: any[] }> {
```